src/lib_ccx/ccx_sub_entry.pb_c.h

```diff
@@ -0,0 +1,86 @@
+/* Generated by the protocol buffer compiler.  DO NOT EDIT! */
```

Comment:
Two questions

1) What does Willem's test suite say about this patch
2) Lots of changes in the code, but none in CHANGES.TXT :-) We need to take
that file seriously... I think each new version is missing new features
that for some reason we haven't documented anywhere.





On Sat, Jun 27, 2015 at 6:02 AM, kisselef <notifications@github.com> wrote:

> In src/lib_ccx/ccx_sub_entry.pb_c.h
> <https://github.com/CCExtractor/ccextractor/pull/192#discussion_r33409345>
> :
>
> > @@ -0,0 +1,86 @@
> > +/* Generated by the protocol buffer compiler.  DO NOT EDIT! */
>
> @anshul1912 <https://github.com/anshul1912> Of course, I agree in
> general, but it is not that case )
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CCExtractor/ccextractor/pull/192/files#r33409345>.
>